src/site/content/en/blog/nfc/index.md

```diff
@@ -0,0 +1,906 @@
+});
+```
+
+Tip: The [cookbook] contains many examples of how to write other types of NDEF
+records.
+
+If an NFC tag should be read while performing a NFC write operation, set the
+`ignoreRead` property to `false` in the options passed to the `write()` method.
+
+```js
+const reader = new NDEFReader();
+reader.scan().then(() => {
+
+  reader.onreading = event => {
+    // TODO: Handle incoming NDEF messages.
+  };
+
+  const writer = new NDEFWriter();
+  return writer.write("Writing data is fun!", { ignoreRead: false });
+
+}).catch(error => {
+  console.log(`Write failed :-( try again: ${error}.`);
+});
+```
+
+If the NFC tag contains an NDEF message that is not meant to be overwritten, it
+is possible to set the `overwrite` property to `false` in the options passed to
+the `write()` method. In that case, the returned promise will reject if an NDEF
+message is already stored in the NFC tag.
+
+```js
+const writer = new NDEFWriter();
+writer.write("Writing data on an empty NFC tag is fun!", { overwrite: false })
+.then(() => {
+  console.log("Message written.");
+}).catch(_ => {
+  console.log(`Write failed :-( try again: ${error}.`);
+});
+```
+
+### Security and permissions {: #security-and-permissions }
+
+The Chrome team has designed and implemented Web NFC using the core principles
+defined in [Controlling Access to Powerful Web Platform
+Features][powerful-apis], including user control, transparency, and ergonomics.
+
+As NFC expands the domain of information potentially available to malicious
+websites, the availability of NFC has been restricted to maximize users'
+awareness and control over NFC use.
+
+<figure class="w-figure">
+  <img src="./nfc-prompt.png" alt="Screenshot of a Web NFC prompt on a website">
+  <figcaption class="w-figcaption">Web NFC user prompt</figcaption>
+</figure>
+
+Web NFC is only available to top-level frames, secure browsing contexts (HTTPS
+only), and origins must first request the `"nfc"` [permission] while handling a
+user gesture (e.g a button click). The NDEFReader `scan()` and NDEFWriter
+`write()` methods trigger a user prompt, if access was not previously granted.
+
+```js
+  document.querySelector("#scanButton").onclick = async () => {
+    const reader = new NDEFReader();
+    // Prompt user to allow website to interact with NFC devices.
+    await reader.scan();
+    reader.onreading = event => {
+      // TODO: Handle incoming NDEF messages.
+    };
+  };
+```
+
+The combination of a user-initiated permission prompt and real-world, physical
+movement of bringing the device over a target NFC tag mirrors the chooser
+pattern found in the other file and device-access APIs.
+
+To perform a scan or write, the web page must be visible when the user touches
+an NFC tag with their device. The browser uses haptic feedback to indicate a
+tap. Access to the NFC radio is blocked if the display is off or the device is
+locked. For backgrounded web pages, receiving and pushing NFC content are
```

Comment:
Same if the screen is off or another app is focused ... maybe make that more explicit

Reply:
When I hear backgrond web pages... I think about tabs... and one not being visible... others might think like me :)

Reply:
Fixed. Thanks!